tests/shiftentercommand.js

```diff
@@ -215,4 +215,94 @@ describe( 'ShiftEnterCommand', () => {
 			} );
 		}
 	} );
+
+	describe( '#isEnabled', () => {
+		test( 'should be disabled if $text cannot be inserted into element',
```

Comment:
Please don't create such `test()` functions when not really necessary. And in this case it doesn't change much – from 5LOC to 4LOC ;)